src/main/java/com/iotiq/application/service/SellerService.java

```diff
@@ -32,10 +46,18 @@ public Seller createDefaultSeller(User currentUser) {
     }
 
     @Transactional
-    public void update(SellerUpdateRequest request){
+    public void update(SellerUpdateRequest request) {
         Seller currentSeller = getCurrentSellerOrCreate();
-        ModelMapperUtil.map(request,currentSeller);
+        ModelMapperUtil.map(request, currentSeller);
 
         sellerRepository.save(currentSeller);
     }
+
+    @Transactional
```

Comment:
Does this need to be transactional?

Reply:
We used getCurrentSellerOrCreate inside the method

Reply:
https://github.com/Iotiq-Erste/kimono-application/issues/48